how-does-it-work.md

```diff
@@ -20,12 +20,12 @@ Using this methodology, it’s expected to be able to start with a stock Linux s
 
 By default, such systems automatically update themselves when a new version of their bootable container image is tagged in the container registry.
 
-Per system state and data are set up per system in /etc and /var per system, and is usually not affected by updates to the bootable container image.
+Per system state and data are set up in /etc and /var per system, and are usually not affected by updates to the bootable container image.
 This includes things like passwords, SSH keys, home directories.
 These things can arrive on the system in the typical way, such as local configuration, cloud-init, or via management tools like Ansible.
 
 It is recommended to have the container image content be read only at runtime for both the operating system and application containers.
-It will be possible and supported to use a local overlay file system as well as a commit process to enable debugging.
+It will be possible and supported to use a local overlay file system as well as a commit process to enable debugging.  However, local changes would not persist across reboots without further action.
```

Comment:
Let's clarify "changes to the local overlay".

Because "local changes" to `/etc` and `/var` are suggested to persist by default (but of course, that's configurable!)

Reply:
Good point.  Fixed!